doc/api/fs.md

```diff
@@ -937,19 +937,20 @@ will be made to remove the destination.
 ```mjs
 import { copyFile, constants } from 'node:fs/promises';
 
+// Example 1: Copy the file, overwriting the destination file if it exists
 try {
   await copyFile('source.txt', 'destination.txt');
-  console.log('source.txt was copied to destination.txt');
-} catch {
-  console.error('The file could not be copied');
+  console.log('File copied successfully: source.txt -> destination.txt');
+} catch (err){
```

Comment:
```suggestion
} catch (err) {
```

Reply:
Thank you for the review. I have made the necessary changes. I will make sure to run a lint check before submitting my next PR.